### ISSUE TITLE
Analysis does not complete

### ISSUE THREAD

**Issue by JoyMitra** [MIT-PAC/droidsafe-src#9]
Hi,

I have been tried to run droidsafe from the command line on a set of benchmarks. Initially I found out that droidsafe throws an error when I run the tool against the apk files that I have. This was because droidsafe is using an older version of apktool. The older version was failing to decompile my apk files. So I update the apktool version. After updating apktool I ran droidsafe on one of the apk files in my benchmark. However, now I see that the analysis does not complete. I let it run for well over ten minutes but it gets stuck at "Starting Generate RCFG...". Can you tell me what is wrong?

**Comment by Anusha-G-Rao** [comment]
Hi, 
Were you able to figure it out? i need some help too

**Comment by JoyMitra** [comment]
@Anusha-G-Rao yes, it works with example apps but the analysis does not complete for all apps. What problems are you facing? Maybe try using apktool 2.2.4 or try setting the variable DROIDSAFE_MEMORY=32 in your environment. Also, run on a machine that has 32GB RAM

**Comment by JoyMitra** [comment]
@mgordon 

I have tried to run droidsafe from the command line on a set of benchmarks. Initially I found out that droidsafe throws an error when I run the tool against the apk files that I have. This was because droidsafe is using an older version of apktool. The older version was failing to decompile my apk files. So I update the apktool version. After updating apktool I ran droidsafe on one of the apk files in my benchmark. However, now I see that the analysis does not complete. I let it run for well over ten minutes but it gets stuck at "Starting Generate RCFG...". I have tried running analysis on the same apk on a machine with 64GB Ram as well, but the analysis does not complete. Can you someone from the team please provide some insight?

**Comment by JoyMitra** [comment]
@Anusha-G-Rao Did you figure it out?
